cardano-config/src/Cardano/TracingOrphanInstances/Network.hs

```diff
@@ -382,22 +386,10 @@ instance (Show peer)
 --
 -- NOTE: this list is sorted by the unqualified name of the outermost type.
 
-instance ( Condense (HeaderHash blk)
-         , Condense (TxId (GenTx blk))
-         , HasHeader blk
-         , HasTxs blk
-         , HasTxId (GenTx blk)
-         )
+instance ( Condense (HeaderHash blk))
       => ToObject (AnyMessage (BlockFetch blk)) where
-  toObject _verb (AnyMessage (MsgBlock blk)) =
-    mkObject
-      [ "kind" .= String "MsgBlock"
```

Comment:
I know information was removed here however, the `HasHeader blk` constraint also applies for serialised blocks. It does not make sense to have a `HasHeader` constraint for a serialized block. I'm not entirely sure how we can get around this.